toolbar.py

```diff
@@ -19,6 +19,8 @@
 from gettext import gettext as _
 import logging
 
```

Comment:
The import re should go here, basically pep8 says you have imports like this:

* built-in
* other
* local

Sugar usually has the simmilar:

* built-in
* gi.repo
* other
*local